src/main.js

```diff
@@ -270,12 +270,20 @@ class MiniGraphCard extends LitElement {
       `;
   }
 
+  getObjectAttr(obj, path) {
+    return path.split('.').reduce((res, key) => res && res[key], obj);
+  }
+
   getEntityState(id) {
     const entityConfig = this.config.entities[id];
     if (this.config.show.state === 'last') {
       return this.points[id][this.points[id].length - 1][V];
     } else if (entityConfig.attribute) {
-      return this.entity[id].attributes[entityConfig.attribute];
+      if (entityConfig.attribute.includes('.')) {
```

Comment:
We should be able to eliminate this `if`, too. If there's no `.` in `attribute`, then `getObjectAttr` will just return `attributes[attribute]`, right?